### PR TITLE
Adds list of ids and optional sort function support

### PR DESCRIPTION
**Pull request by mparke**
precursor to changes in #1191

### CODE REVIEW

File: src/api/apiResultActionReducerGenerator.js [hunk 1]
```diff
@@ -118,8 +119,15 @@ export class ReducerGenerator {
         resource: oneObject,
       }), oldState);
 
+    let ids = Object.values(newState[config.plural]).map((obj) => obj.id);
+
+    if (config.sortFn) {
+      ids = config.sortFn(ids, newState[config.plural]);
```
Comment (eatonphil) [inline src/api/apiResultActionReducerGenerator.js]: Personally, I prefer full names for things (sortFn): sort or sortFunction. Not a blocker.

Comment (mparke) [inline src/api/apiResultActionReducerGenerator.js]: agreed. I have a habit of trying not to use reserved words in var names, fwiw

Comment (eatonphil) [inline src/api/apiResultActionReducerGenerator.js]: Is sort a reserved name?

Comment (mparke) [inline src/api/apiResultActionReducerGenerator.js]: lol. oops

**Pull request merged**